android/src/main/java/nl/fhict/happynews/android/manager/PostManager.java

```diff
@@ -155,4 +158,20 @@ public Date deserialize(JsonElement json, Type typeOfT, JsonDeserializationConte
             }
         });
     }
+
+    /**
+     * Send a call to the api to flag given post with a given reason.
+     *
+     * @param postUuid uuid for the post
+     * @param reason   reason for flag
+     */
+    public void flagPost(Context context,String postUuid, String reason) {
+        String uri = apiUrl + "/post/" + postUuid + "/flag";
+        JsonObject json = new JsonObject();
+        json.addProperty("reason", reason);
+        Ion.with(context)
+            .load(uri)
+            .setJsonObjectBody(json)
+            .asJsonObject();
```

Comment:
I think it would be nice to provide some feedback to the user, perhaps a `Toast` or a `SnackBar`?